Formula/perunDev.rb

```diff
@@ -0,0 +1,29 @@
+class Perun < Formula
```

Comment:
Please rename the file to use only lower case (perundev.rb) and I believe the class name here is important to be Perundev then ;)